test/cmdlineTests/standard_import_evmasm/input.json

```diff
@@ -0,0 +1,417 @@
+            {
+              "begin": 539,
+              "end": 568,
+              "name": "SSTORE",
+              "source": 0
+            },
+            {
+              "begin": 586,
+              "end": 601,
+              "name": "PUSH #[$]",
+              "source": 0,
+              "value": "000000000000000000000000000000000000000000000000ffffffffffffffff"
+            },
+            {
+              "begin": 582,
+              "end": 584,
+              "name": "PUSH",
+              "source": 0,
+              "value": "F"
+            },
+            {
+              "begin": 575,
+              "end": 602,
+              "name": "SSTORE",
+              "source": 0
+            }
+          ],
+          ".data": {
+            "0": {
+              ".code": [
+                {
+                  "begin": 658,
+                  "end": 673,
+                  "name": "PUSH [$]",
+                  "source": 0,
+                  "value": "0000000000000000000000000000000000000000000000000000000000000000"
+                },
+                {
+                  "begin": 654,
+                  "end": 656,
+                  "name": "PUSH",
+                  "source": 0,
+                  "value": "10"
+                },
+                {
+                  "begin": 647,
+                  "end": 674,
+                  "name": "SSTORE",
+                  "source": 0
+                },
+                {
+                  "begin": 694,
+                  "end": 707,
+                  "name": "PUSH #[$]",
+                  "source": 0,
+                  "value": "0000000000000000000000000000000000000000000000000000000000000000"
+                },
+                {
+                  "begin": 690,
+                  "end": 692,
+                  "name": "PUSH",
+                  "source": 0,
+                  "value": "11"
+                },
+                {
+                  "begin": 683,
+                  "end": 708,
+                  "name": "SSTORE",
+                  "source": 0
+                }
+              ],
+              ".data": {
+                "0": {
+                  ".code": [
+                    {
+                      "begin": 761,
+                      "end": 770,
+                      "name": "INVALID",
+                      "source": 0
+                    }
+                  ]
+                }
+              }
+            },
+            "1": {
+              ".code": [
+                {
+                  "begin": 833,
+                  "end": 842,
+                  "name": "INVALID",
+                  "source": 0
+                }
+              ]
+            }
+          }
+        },
+        "ACAF3289D7B601CBD114FB36C4D29C85BBFD5E133F14CB355C3FD8D99367964F": "48656c6c6f2c20576f726c6421"
+      },
+      "sourceList": [
+        "<stdin>"
+      ]
+    }
+  },
+  "settings": {
+    "outputSelection": {
+      "*": {
+        "": [
+          "evm.assembly",
```

Comment:
Still need to check for actual valid values in the context of `EvmAssemblyJSON`: error messages...

Reply:
Maybe we could just leave it as is.. its just experimental 😅